i18n/uk.toml

```
@@ -0,0 +1,88 @@
+
+  [links.close-menu]
+  other = "Закрити меню"
+
+  [links.documentation]
+  other = "Документація"
+
+  [links.join-the-discussion]
+  other = "Долучитись до дискусії"
+
+[socials]
+  [socials.twitter]
+  other = "Twitter"
+
+  [socials.reddit]
+  other = "Reddit"
+
+  [socials.forums]
+  other = "Forums"
+
+  [socials.slack]
+  other = "Slack"
+
+  [socials.github]
+  other = "Github"
+
+[copyright]
+  [copyright.line1]
+  other = "© 2020 Проект Rocky Linux. Всі права застережено."
+
+  [copyright.line2]
+  other = "CentOS є зареєстрованою торговою маркою Red Hat, Inc. Проект Rocky Linux не афілійований з Red Hat, Inc. та не схвалений Red Hat, Inc."
+
+[errors]
+  [errors.404]
+    [errors.404.title]
+    other = "Сторінку не знайдено."
+
+    [errors.404.explanation]
+    other = "Запитуваний ресурс не було знайдено. Будь ласка спробуйте ще раз."
+
+[landing]
+  [landing.p1.heading]
+  other = "Rocky Linux"
+
+  [landing.p1.content]
+  other = "Спільна ініціатива зі створення системи Linux коропортативного рівня."
+
+  [landing.p2.heading]
+  other = "Що таке проект Rocky Linux?"
+
+  [landing.p2.content]
+  other = "Rocky Linux це спільна операційна система корпоративного рівня, спроектована, щоб забезпечити 100% сумісності з найпопулярнішою американською корпоративною Linux системою CentOS. Розробка Rocky Linux була спровокована кардинальною зміною у напрямку розвитку CentOS. Rocky Linux активно розвивається спільнотою. Проектом керує Gregory Kurtzer, засновник CentOS. Наразі, планова дата першого випуску невідома. Бажаючі долучитись, будь ласка виходьте на зв'язок через один з каналів зазначених на цьому сайті."
+
+[faq]
+  [faq.title]
+  other = "Часті запитання."
+
+  [faq.q]
+    [faq.q.downstream-partner-direction]
+    other = "Що означає, \"кардинальна зміна у напрямку розвитку CentOS?\""
+
+    [faq.q.come-in]
+    other = "Отже, яке місце Rocky Linux на ринку?"
+
+  [faq.a]
+    [faq.a.downstream-partner-direction]
+    other = "Проект CentOS нещодавно оголосив про зміну стратегії розвитку та підтримки операційної системи CentOS. Попередньо, CentOS існувала як безкоштовний дистрибутив комерційної та стабільної системи (CentOS отримувала правки та оновлення після того, як їх отримав комерційний дистрибутив). Надалі, CentOS стане гілкою розробки та тестування комерційного дистрибутиву, а отже тестові правки та оновлення будуть виходити для CentOS до того, як потраплять в комерційний дистрибутив. До того ж, період підтримки CentOS Linux 8 було скорочено, з 31-го травня, 2029-го року до 31-го грудня, 2021-го."
+
+    [faq.a.come-in]
+    other = "Метою Rocky Linux є побудова безкоштовного дистрибутиву, випуски якого будуть виходити вслід за випусками комерційної системи, а не перед ними. Роль такого дистрибутиву раніше виконував CentOS."
+    other = "Rocky Linux aims to function as a downstream build as CentOS had done previously, building releases after they have been added by the upstream vendor, not before."
```

```suggestion
    other = "Метою Rocky Linux є побудова безкоштовного дистрибутиву, випуски якого будуть виходити вслід за випусками комерційної системи, а не перед ними. Роль такого дистрибутиву раніше виконував CentOS."
```

Let's make sure we remove this duplicate key in English.